fetch.js

```diff
@@ -236,6 +236,21 @@
       }
     }
 
+    if (support.arrayBuffer) {
+      this.arrayBuffer = function() {
+        if (this._bodyArrayBuffer) {
+          var rejected = consumed(this)
+          if (rejected) {
+            return rejected
+          }
+
+          return Promise.resolve(this._bodyArrayBuffer.buffer)
+        } else if (support.blob) {
```

Comment:
Indeed, it seems that `ArrayBuffer` is more widely supported than `Blob`. Fixed this.